src/main/java/CoffeeHouse.java

```diff
@@ -0,0 +1,133 @@
+import client.Client;
+import coffeeMaker.Barista;
+import coffeeMaker.CoffeeMachine;
+import coffeeMaker.Coffeemaker;
+import payment.CardPayment;
+import payment.CashPayment;
+import storage.Storage;
+import utils.Coffee;
+import utils.Telemetry;
+
+import java.util.Random;
+
+class CoffeeHouse {
+    private Storage storage = Storage.getInstance();
+    private Random random = new Random();
+    private Telemetry telemetry = new Telemetry();
+
+    void pedal() {
+        while (true) {
+            Client client = new Client();
+            System.out.println();
+            Coffee coffee = Coffee.getCoffee();
+            if (!hasIngredients(coffee)) {
+                System.out.println("We cannot make coffee today, CoffeeHouse is closed");
+                System.out.println(telemetry.toString());
+                break;
+            } else {
+                System.out.println(client.getName() + " " + (client.isHasCard() ? "card" : "cash") + " "
+                        + client.getAmountOfCash() + " " + client.getAmountOfMoneyOnTheCard());
+                //payment(clients[i], (client) -> {});
+                if (!payment(client, coffee)) {
+                    System.out.println("The customer doesn't have enough money");
+                } else {
+                    makeCoffee(coffee, client);
+                }
+            }
+        }
+    }
+
+    private boolean hasIngredients(Coffee coffee) {
+        switch (coffee) {
+            case LATTE://2c 2m 1s
+                System.out.println("Latte was chosen");
+                return storage.isCoffee(1) && storage.isMilk(2) && storage.isCup(1);
```

Comment:
Recept should be in Recepi card. For avoiding using "magic numbers" and to have one source of truth